app/views/layouts/sumofus.slim

```diff
@@ -14,5 +14,7 @@ html
     = yield
 
     .mobile-indicator
-    - if current_user && @page.present?
-      = link_to 'Edit Page', edit_page_path(@page), class: 'campaigner-overlay__edit-link'
+    - if @page.present?
+      span.hidden.page_id = @page.id
```

Comment:
This is almost certainly the wrong way to do this, but we need the page ID to be available to the Braintree API so that we can send the information to ActionKit, so thoughts on how to effectively make that happen are welcome.